app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestConfig.html

```diff
@@ -1,11 +1,47 @@
-<div ng-if="ctrl.state.loaded">
+<div ng-if="ctrl.state.loaded" xmlns="http://www.w3.org/1999/html">
   <div class="alert alert-warning">
     <strong>Warning!</strong> This stage is under active development and is subject to change.
   </div>
-  <v2-wizard-page key="basicSettings" label="Basic Settings" mark-complete-on-view="false">
-    <kubernetes-manifest-basic-settings command="ctrl.$scope.stage" metadata="ctrl.metadata"> </kubernetes-manifest-basic-settings>
+
+  <v2-wizard-page key="basicSettings" label="Basic Settings">
```

Comment:
I don't love the `v2-wizard-page` being used here. Are there other examples of it being used in a stage config? Even if it renders OK now, if anyone ever refactors that component, they could easily break this stage.

Reply:
SGTM